distribution/sender/routes/database-event-reader.xml

```diff
@@ -0,0 +1,90 @@
+<routes xmlns="http://camel.apache.org/schema/spring">
```

Comment:
Rename this to db-event-handler-route.xml, route names needs to end with **-route.xml**

Reply:
handler or reader??